src/bank00.asm

```diff
@@ -9006,9 +9006,24 @@ getDialogTextInsertionPoint:
     ld   A, [wWindowTextSpaceLeftOnLine]               ;; 00:375f $fa $ba $d8
     ld   C, A                                          ;; 00:3762 $4f
     ret                                                ;; 00:3763 $c9
-    db   $d5, $3e, $7f, $15, $cd, $44, $38, $14        ;; 00:3764 ????????
-    db   $3e, $7f, $cd, $44, $38, $1c, $05, $20        ;; 00:376c ????????
-    db   $f0, $d1, $c9                                 ;; 00:3774 ???
```

Comment:
Just looked at what this is to make sure it's not replacing something potentially valuable (and since I'm looking for 47 bytes to replace somewhere). It looks like it's designed to blank two adjacent tiles and then increment to the next line.   Seems fine to replace. There's actually several functions around setting (and even getting) things in VRAM that look like they got replaced by `storeTileAatScreenPositionDE`.

Here's the function for posterity:

```
call_00_3764:
    push DE
.loop:
    ld A, $7f
    dec D
    call storeTileAatDialogPositionDE
    inc D
    ld A, $7f
    call storeTileAatDialogPositionDE
    inc E
    dec B
    jr NZ, .loop
    pop DE
    ret
```